### ISSUE TITLE
Bad UX in documentation banner images `

### ISSUE THREAD

**Issue by tom-ricci** [traefik/traefik#10886]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### The Problem

I'm trying to learn about Traefik through the docs, and I've found the banner images (such as this one):
![https://github.com/traefik/traefik/assets/79022795/194c7d73-6474-459c-bd60-9617a09d3bc6](https://github.com/traefik/traefik/assets/79022795/658792c0-68ac-419c-9d54-cad8f9d2d71a)
to be hard to understand from the perspective of someone who's just getting started with the proxy. 

The (relatively) complex geometry of the graph makes it confusing; I'm finding myself focusing more on the arrows themselves than I am the actual content of the graph, and I feel it makes the images largely redundant.

I think a basic graph would be better, like this:
![Untitled](https://github.com/traefik/traefik/assets/79022795/06a8e75f-9d81-4433-b026-44ce4734e726)
The topology is clear and intuitive, and allows readers to immediately understand the connections instead of thinking about them too much.

**Comment by nmengin** [comment]
Hey @tom-ricci,

Thanks for your suggestion.

We are interested in this issue, but we’re unsure about the use case and the traction it will receive. We are going to leave the status as kind/proposal to give the community time to let us know if they would like this idea.

We will reevaluate as people respond.

Conversation is time-boxed to 6 months.

**Comment by VisheshBansal** [comment]
I agree the arrows are bit distracting (found them myself) and a simple clean diagram might be better helpful.

At the same time I would prefer the information such as
```
Address: 8081
FORWARDED HEADER
```
in the original banner since it is can be easy to overlook the http:// and https:// in a first glance, so would still like those in the respective entrypoint rectangles.

**Comment by kevinpollet** [comment]
Hello @tom-ricci,

This proposal was opened 6 months ago with almost no traction.
We won't add it to the Roadmap yet and we prefer to close it.

We'll re-open it later if necessary.
Thank you @tom-ricci for the suggestion.
